repose-aggregator/components/filters/rate-limiting-filter/src/test/java/org/openrepose/filters/ratelimiting/SchemaTest.java

```diff
@@ -87,19 +87,5 @@ public void shouldValidateAgainstLiveLimitsExample() throws Exception {
             assertThat("Expected element should have child elements",
                     jaxbUnmarshaller.unmarshal(sampleSource, Limits.class).getValue().getRates().getRate().size(), greaterThan(0));
         }
-
-        @Test
-        @Ignore
-        public void output() throws Exception {
```

Comment:
Also not necessary. This isn't actually a test (which is probably why it was ignored). It just prints an example as both XML and JSON. Considering it was brought in with the very first commit, I don't see any value here. You could argue that it tests that transformer, but that really isn't the job of this test. Furthermore, the transformer methods called here are not called anywhere else. So really, I should go delete those methods. Maybe that entire class? I'll try that and see if anything breaks.